protected/views/site/help.php

```diff
@@ -1,362 +1,929 @@
+                    <p>Optional information includes links to additional resources and related manuscripts, accessions for data in other databases (prefixes are found in the Links tab), and <a href="http://gigadb.org/site/help#relation" target="_blank">relationship</a> (if any) to a previously published <em>Giga</em>DB dataset (selected from a controlled vocabulary list).</p>
+                    <h2 class="h5">Samples</h2>
+                    <p>Required information includes a sample ID or name (please use an <a href="http://www.ncbi.nlm.nih.gov/biosample" target="_blank">NCBI BioSample ID</a> when possible), species <a href="http://www.ncbi.nlm.nih.gov/Taxonomy" target="_blank">NCBI taxonomy ID</a>, and species common name.</p>
+                    <p>Optional information includes sample attributes (these are automatically populated in <em>Giga</em>DB if an <a href="http://www.ncbi.nlm.nih.gov/biosample" target="_blank">NCBI BioSample ID</a> is provided).</p>
+                    <h2 class="h5">Files</h2>
+                    <p>Required information includes a file name or path relative to your home directory and <a href="http://gigadb.org/site/help#filetype" target="_blank">file type</a> (selected from a controlled vocabulary list). A readme file must be provided.</p>
+                    <p>Please note;<br>
+                        -Filenames should be unique. <br>
+                        -Filenames should not include spaces. We recommend using the underscore (_) in place of spaces in the filenames.<br>
+                        -Filenames should only include the following characters a-z,A-Z,0-9,_,-,+,. </p>
+                    <p>Optional information includes a file description and a sample ID or name.</p>
+                    <p><strong>Step 3</strong> - confirm you have read our <a href="http://gigadb.org/site/term/" target="_blank">Terms of use</a> page and upload the completed Excel template file.</p>
+                    <p>You can expect a response from the <a href="http://gigadb.org/" target="_blank"><em>Giga</em>DB</a> team within 5 days to verify the information in your submission and to arrange upload of your files to our FTP site.</p>
+                    <p>If you have any questions, please contact us at <a href="mailto:database@gigasciencejournal.com" target="_blank">database@gigasciencejournal.com</a>.</p>
+                </div>
+                <div role="tabpanel" class="tab-pane" id="vocabulary" aria-labelledby="livocabulary">
+                    <section aria-labelledby="datasettypes" class="m-0">
+                        <h2 class="page-subtitle" id="datasettypes">Dataset types</h2>
+                        <dl class="help-description-list">
```

Comment:
this was a sequence of `<p>`, updated it to a more semantic `<dl>` (same goes for all `<dl>` that follow)